seedcase_sprout/core/create_package_properties.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+from seedcase_sprout.core.create_properties_path import create_properties_path
+from seedcase_sprout.core.properties import PackageProperties
+from seedcase_sprout.core.sprout_checks.check_package_properties import (
+    check_package_properties,
+)
+from seedcase_sprout.core.write_json import write_json
+
+
+def create_package_properties(
+    properties: PackageProperties, path: Path = Path.cwd()
+) -> list[Path]:
+    """Creates a new `datapackage.json` file with properties for a new package.
+
+    This is the first function to use to create a new data package. It creates a
+    `datapackage.json` file based on the values you give it with the `properties`
+    argument. You can use the `PackageProperties` class to help create a valid
+    properties object that has the correct fields.
+
+    By default, the function will add the `id`, `version`, and `created` fields to the
+    properties object. See the `PackageProperties` class documentation for more
+    information.
+
+    Args:
+        properties: A `PackageProperties` object containing the properties for the
+            package.
+        path: The path to the folder where the data package will be created.
+            Defaults to the working directory.
+
+    Returns:
+        A path to the created folder and `datapackage.json` file.
```

Comment:
```suggestion
        The path to the `datapackage.json` file in the created folder.
```
To make it clear that it returns one path